gcovr/tests/test_args.py

```diff
@@ -114,6 +115,41 @@ def test_non_existing_directory_csv(capsys):
     helper_test_non_existing_directory_output(capsys, '--csv')
 
 
+def helper_test_non_existing_directory_2_output(capsys, option='--option'):
```

Comment:
Default value is misleading here.

```suggestion
def helper_test_non_existing_directory_2_output(capsys, option):
```

Reply:
Oh, tht's an error, the default must be `'--output'` as in
```python
def helper_test_non_existing_directory_output(capsys, option='--output'):
```